css/main.css

```diff
@@ -0,0 +1,82 @@
+body {
+  font-family: Helvetica, Arial, sans-serif;
+  margin: 0;
+  min-height: 100vh;
+}
+
+header {
```

Comment:
It doesn't need to be 100% identical to other demos but included a picture to show what I mean:
<img width="1029" alt="CleanShot 2022-01-03 at 16 04 30@2x" src="https://user-images.githubusercontent.com/18265845/147980354-a0706159-cd8e-4bff-8b3d-bc0c93c0e508.png">


I think it would be good to vertically align the header content (esp. the right side), and reduce the logo size a bit. Might be easier with flex-box but totally your call.